app/models/api/commodity.rb

```diff
@@ -48,8 +48,28 @@ def applicable_measures
       @applicable_measures ||= no_additional_code_measures + additional_code_measures
     end
 
+    def applicable_excise_measure_units
+      applicable_measure_units.each_with_object({}) do |(unit, value), acc|
```

Comment:
Presumably `acc` is an abbreviation for accumulator - full name would be easier to read though?

Reply:
I'll rename this separately since it's the name we have historically used for the accumulator.